app/controllers/dishes_controller.rb

```diff
@@ -1,27 +1,44 @@
 # frozen_string_literal: true
 
 class DishesController < ApplicationController
-  before_action :dish_params, only: %i[create index]
-  before_action :set_restaurant, only: %i[new create index]
-
+  before_action :authenticate_user!
```

Comment:
let's set this method in application_controller and set method skip_validation only in the controller that should skip that instead of adding this line in each controller